addons/website/static/src/snippets/s_social_media/000.scss

```diff
@@ -66,5 +66,10 @@
                 @extend .text-tiktok;
             }
         }
+        .s_social_media_discord {
+            &, &:hover, &:focus {
+                @extend .text-discord;
+            }
+        }
```

Comment:
Hello @Polymorphe57 since i have implemented these changes in the file , I'm encountering this check in my current build:  https://runbot.odoo.com/runbot/batch/1552234/build/65978081. Should we ignore this issue, or should I implement a mixin method for all cases in this particular file ?
cc @msh-odoo